_security/access-control/api.md

```diff
@@ -36,6 +36,22 @@ plugins.security.restapi.endpoints_disabled.<role>.<endpoint>: ["<method>", ...]
 ```
 {% include copy.html %}
 
+Roles also allow you to control access to specific REST APIs. You can add individual or multiple cluster permissions to a role and grant users access to associated APIs when they are mapped to the role. The following list of cluster permissions includes the endpoints that correspond to the Security REST APIs:
+
+| **Permission**                 | **APIs Granted**                 | **Description**                                                                                                                                   |
+|:-------------------------------|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------|
+| restapi:admin/actiongroups     | `/actiongroup` & `/actiongroups` | Permission to get, delete, create, and patch actions groups (including bulk updates)                                                              |
+| restapi:admin/allowlist        | `/allowlist`                     | Permission to add any endpoints and HTTP requests to a list of allowed endpoints and requests.                                                    |
+| restapi:admin/internalusers    | `/internaluser` & `/user`        | Permission to add, retrieve, modify and delete any user in the cluster                                                                            |
```

Comment:
It's a hassle. But we follow the "Oxford comma" rule: a comma between all elements of a list. As below.
```suggestion
| restapi:admin/internalusers    | `/internaluser` & `/user`        | Permission to add, retrieve, modify, and delete any user in the cluster.                                                                            |
```
Global comment.